packages/zoe/docs/zoe-catalog-of-storage.md

```diff
@@ -0,0 +1,217 @@
+# Catalog of Zoe Maps/WeakMaps/Stores
+
+## Zoe Service (one per chain)
+
+### installations - native WeakSet
+
+An installation is added whenever a user calls `E(zoe).install` to
+install a contract bundle. 
```

Comment:
Can one add the same installation more than once?   (It's okay to respond that this document doesn't want to cover that kind of issue.)

Reply:
I think I'll leave it out this doc, but there's a comment in the code in the refactoring that covers this! The same bundle can be installed more than once, but it will get a new installation handle.